pulp_container/tests/functional/api/test_pull_content.py

```diff
@@ -70,10 +71,23 @@ def setUpClass(cls):
 
         delete_orphans()
 
+        # defining a shared path will help us to identify whether namespaces and
+        # matching distributions are correctly determined from within the plugin
+        random_dist_path = str(uuid.uuid4())
```

Comment:
I was reviewing this PR and remembered we used to have this limitation https://github.com/pulp/pulp_container/issues/490
Now it is no longer there, so I am going to close it